easybuild/easyconfigs/l/libxml2/libxml2-2.9.3-foss-2015a-Python-2.7.11.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
```

Comment:
for libxml2 build that include a Python dep, the existing libxml2 easyblock should be used